src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ManagedGroupService.scala

```diff
@@ -19,12 +19,14 @@ class ManagedGroupService(private val resourceService: ResourceService, private
   def managedGroupType: ResourceType = resourceTypes.getOrElse(ManagedGroupService.managedGroupTypeName, throw new WorkbenchException(s"resource type ${ManagedGroupService.managedGroupTypeName.value} not found"))
   def memberRole = managedGroupType.roles.find(_.roleName == ManagedGroupService.memberRoleName).getOrElse(throw new WorkbenchException(s"${ManagedGroupService.memberRoleName} role does not exist in $managedGroupType"))
 
+  // TODO: What should be returned here?  Just a Resource object?  The report generated from the sync with Google?  Both?
```

Comment:
I don't think we want the google stuff

Reply:
Besides, I don't think we want to wait for the sync to complete before returning.